packages/admin-ui-docs/docs/experimental/inline.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Inline
+path: /inline/
+---
+
+# Inline
+
+Render a set of components in a row with equal spacing between them. It also supports wrapping.
```

Comment:
This does the same as the `Set` in the horizontal mode, right?

Reply:
Yes! Our set is weird because Stacks should not wrap (or being horizontal)